setup.py

```diff
@@ -1,38 +1,32 @@
-import sys
+import pathlib
+import re
 
-from Cython.Build import cythonize
 from setuptools import find_packages, setup
-from setuptools.extension import Extension
-
-import versioneer
-
-_libraries = ["archive_and_deps"]
-if sys.platform == "win32":
-    _libraries.append("advapi32")
-    _libraries.append("user32")
-archive_utils_cy_extension = Extension(
-    name="conda_package_handling.archive_utils_cy",
-    sources=["src/conda_package_handling/archive_utils_cy.pyx"],
-    libraries=_libraries,
-)
 
+version = (
+    re.search(
+        r'__version__\s+=\s+"(.*)"',
+        pathlib.Path("src/conda_package_handling/__init__.py").read_text(),
+    )
+    or []  # typing
+)[1]
```

Comment:
disagree

Reply:
@dholth , manually parsing a python file with a regex like this is not ideal. We should use a more robust solution which is what I believe @jezdez is suggesting. Please reconsider this or justify your position with a more meaningful response.

Reply:
I have used setuptools-scm before and I think it is a pain to do. For example it can fail to call the revision control system, and you have to wait for the revision control system to do its work.